lutris/installer/interpreter.py

```diff
@@ -135,7 +135,7 @@ def __init__(self, installer, parent):
         self.game_name = self.script.get("custom-name") or installer["name"]
         self.game_slug = installer["game_slug"]
         self.steamid = installer.get("steamid")
-        self.gogid = installer.get("gogid")
+        self.gogid = self.script["game"]["gogid"] or installer.get("gogid")
```

Comment:
Not sure if `game` is the best place in the installer YAML to place the optional `gogid` field, so I'm open to better suggestions.

Reply:
we put appid for steam in the `games` section so this is consistent with our current behavior.